packages/__docs__/src/Playground/index.tsx

```diff
@@ -58,26 +59,10 @@ const fullScreenIconPath = (
 /* eslint-enable max-len */
 
 @withStyle(generateStyle, generateComponentTheme)
-class Playground extends Component {
-  static propTypes = {
-    // eslint-disable-next-line react/require-default-props
-    makeStyles: PropTypes.func,
-    // eslint-disable-next-line react/require-default-props
-    styles: PropTypes.object,
-    title: PropTypes.string.isRequired,
-    code: PropTypes.string.isRequired,
-    language: PropTypes.string.isRequired,
-    render: PropTypes.bool,
-    background: PropTypes.oneOf([
-      'checkerboard',
-      'checkerboard-inverse',
-      'light',
-      'inverse',
-      'none'
-    ]),
-    readOnly: PropTypes.bool
-  }
-
+class Playground extends Component<PlaygroundProps, PlaygroundState> {
+  _fullScreenButton: IconButton | null | undefined
```

Comment:
We try to type the variables in a way that they cannot be both `null` and `undefined`. You can do this if you set the default value to `null`:

`_fullScreenButton: IconButton | null = null`

(Also, please move it under the static props, it's hard to find hiding up here :) )